### DISCUSSION TITLE
General thoughts on DuckDuckGo's desktop browsers (Windows/MacOS)

### DISCUSSION THREAD

**Discussion by Ja-ngolo** [privacytests/privacytests.org#168] [General]
So I have seen a lot of marketing buzz around the new duckduckgo desktop browsers. I'm not so sure what they really bring to the table from a privacy perspective given we have good alternatives like Mullvad, Librewolf and Brave. It would be interesting to know what others think, and if it would be worth it to maybe include in the comparisons.

**Comment by dtdan-03** [comment]
I agree with that, but for me, it would also be interesting to see the DuckDuckGo mobile app (in my case Android) next to the desktop version in the comparisons
